tests/test_basis.py

```diff
@@ -89,3 +89,16 @@ def runTest(self):
 class TestFacetExpansionHexS2(TestFacetExpansion):
     mesh_type = MeshHex
     elem_type = ElementHexS2
+
+
+class TestDofsMerge(TestCase):
+
+    def runTest(self):
+        m = MeshTri()
+        basis = InteriorBasis(m, ElementTriP2())
+        D1 = basis.get_dofs(lambda x: x[0] == 0)
+        D2 = basis.get_dofs(lambda x: x[0] == 1)
+        D3 = basis.get_dofs(lambda x: x[1] == 1)
+        D4 = basis.get_dofs(lambda x: x[1] == 0)
+        assert_allclose(D1 | D2 | D3 | D4,
+                        basis.get_dofs())
```

Comment:
I thought `.get_dofs` was to be deprecated and replaced by `.find_dofs`, as in #367?

Reply:
Yeah I thought so too but now looking at the current implementation it's pretty short and concise. Also `find_dofs` wasn't the ultimate solution as I thought it would be: we still need something like `find_subdomain_dofs` or `get_subdomain_dofs` to find DOF's corresponding to the subdomains. So still further improvements required on this regard.

Eventually I'd like to have something like
```
x, D1 = basis.set_boundary_condition(lambda x: x[0] == 0, 5.0)
x, D2 = basis.set_boundary_condition(lambda x: x[0] == 1, lambda x: x[0] ** 2, x)
...
```
which builds on top of the existing more lower level functionality, whatever that'll be.

Reply:
O. K.; interesting, thanks.